### DISCUSSION TITLE
Firefox:  9.0.1.22a1 clicking on the icon results in blank box

### DISCUSSION THREAD

**Discussion by frogandspanner** [aclap-dev/video-downloadhelper#656] [Help and Q&A]
### Your question

Yesterday (24th May 2024). Firefox automatically installed 9.0.1.22a1 (BETA)
now when I click the VDH icon I am presented with a blank box.

I have purged may cache and jistory and restarted, but get the same result.

### Addon & CoApp version and details

As stated above.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I would expect that reporting your operating system & its version as well as your version of Firefox might help @paulrouget Paul to diagnose your problem better.  I have to say that with Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19, I am not seeing this problem.

**Comment by paulrouget** [comment]
@frogandspanner Can you do this please: https://github.com/aclap-dev/video-downloadhelper/wiki/How-to-enable-logs - _but please remove your email and license for the logs when you share them_.

And which version of Firefox are you using? We only compatible with Version 113 and above.

**Reply by GrampaWildWilly** [reply]
Have you fixed that yet so it doesn't put the user's license number in the output for unsuspecting user's to publish licenses for all the world to steal?  And while you're at it, don't put the E-mail address in there either,

**Reply by paulrouget** [reply]
Yes - that's coming in in v9.0.2.1.

**Comment by slonech** [comment]
I am seeing the same issue, but it appears to be random. Sometimes it fills the dialog box and works perfectly and other times it is blank.
Currently running Windows 10, 64 bit and Firefox version 127.0 ,64 bit.  Showed the same issue on previous Firefox version.  Currently running version 9.0.2.3a1 of Video DowloadHelper.

**Reply by lilmnm-kamikaze-** [reply]
Same for me as well. But on version 2.0.2.3
